src/cplscheme/BaseCouplingScheme.hpp

```diff
@@ -403,11 +397,27 @@ class BaseCouplingScheme : public CouplingScheme {
    */
   bool accelerate();
 
+  /**
+   * @brief stores current data in buffer of Waveform
+   */
+  void storeWindowData();
+
   /**
    * @brief Extrapolate coupling data from values of previous time windows
-   * @param data Data fields to extrapolate
    */
-  void extrapolateData(DataMap &data);
+  void extrapolateData();
+
+  /**
+   * @brief used for storing send/receive data at end of acceleration, if not converged.
+   */
+  void storeLastIteration() // @todo: Try to make this private. This method is currently, only used for sake of testing.
```

Comment:
Why only for testing? You call it in `accelerate`.

Reply:
This todo is outdated. I removed it in c6e1deb407f3bbb7cdd0b84c9f71cbe156c3c958 and also renamed `storeLastIteration` to `storeIteration`.